node/hybrid-node-tests/Ecma-test/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "ecma-test",
+    "version": "1.0.0",
+    "description": "",
+    "main": "Ecma-test.mjs",
+    "type": "module",
+    "scripts": {
+        "build-before-test": "cd ../../../node && npm run build",
+        "test-package-for-ecma": "npm run build-before-test && node Ecma-test.mjs",
+        "test-for-ci": "npm run test-package-for-ecma",
```

Comment:
why repeat the script?

Reply:
Cause I'm lazy and want one script doing all, but ill change it to a more logical way.

Reply:
Meant something else, will fix